addons/pinniped/post-deploy/pkg/configure/configure_test.go

```diff
@@ -261,6 +277,50 @@ func TestPinniped(t *testing.T) {
 				kubetesting.NewRootGetAction(jwtAuthenticatorGVR, jwtAuthenticator.Name),
 				kubetesting.NewRootUpdateAction(jwtAuthenticatorGVR, jwtAuthenticator),
 			},
+			wantJWTAuthenticator: authv1alpha1.JWTAuthenticator{},
+		},
+		{
+			name: "workload cluster configured from scratch with explicit concierge audience",
+			newKubeClient: func() *kubefake.Clientset {
+				return kubefake.NewSimpleClientset()
+			},
+			newCertManagerClient: func() *certmanagerfake.Clientset {
+				return certmanagerfake.NewSimpleClientset(supervisorCertificate)
+			},
+			newSupervisorClient: func() *pinnipedsupervisorfake.Clientset {
+				return pinnipedsupervisorfake.NewSimpleClientset()
+			},
+			newConciergeClient: func() *pinnipedconciergefake.Clientset {
+				// The jwtauthenticator is usually deployed onto the workload cluster with all fields set
+				// correctly except for the audience; see:
+				//   https://github.com/vmware-tanzu/community-edition/blob/1aa7936d88f5d9b04398d800bfe83a165619ee16/addons/packages/pinniped/0.4.4/bundle/config/overlay/pinniped-jwtauthenticator.yaml
+				jwtAuthenticatorWIthUUID := jwtAuthenticatorWithUUIDAudience.DeepCopy()
+				return pinnipedconciergefake.NewSimpleClientset(jwtAuthenticatorWIthUUID)
+			},
+			parameters: Parameters{
+				ClusterType:              "workload",
+				ClusterName:              jwtAuthenticatorWithUUIDAudience.Spec.Audience,
```

Comment:
shouldn't this be `JWTAuthenticatorAudience:` instead of `ClusterName` to test the new audience logic?

Reply:
Oop, yup lemme swap that.